User/Include/UserPcd.h

```diff
@@ -35,6 +35,7 @@
 
 #include <Uefi.h>
 #include <Library/PcdLib.h>
+#include <Library/DebugLib.h>
```

Comment:
Please do not include this header for all files. OcMp3Lib build failure most likely happens due to this. Move this to UserPcd.c.

Reply:
Corrected